### ISSUE TITLE
A huge gap when train this model with batchsize=16 on 4 GPUs

### ISSUE THREAD

**Issue by cuizhenyu1995** [guxinqian/Simple-CCReID#3]
When I tried to train this model on 4 GPUs, I set the batchsize=16, which may be the same with batchsize=32 on 2 GPUs. However, there is a huge performance decay on performance, which worried me a lot. May I get any advise?

**Comment by guxinqian** [comment]
Batchsize=32 is optimal setting for BatchNorm in re-id task, especially for the small-scale datasets.
